src/js/Mixins/Snapping.js

```diff
@@ -212,15 +212,19 @@ const SnapMixin = {
         layer instanceof L.Polyline ||
         layer instanceof L.Marker ||
         layer instanceof L.CircleMarker
+
       ) {
-        layers.push(layer);
+          if(layer.options.snapIgnore !== true){
+              layers.push(layer);
+          }
+
 
         // this is for debugging
         const debugLine = L.polyline([], { color: 'red', pmIgnore: true });
         debugIndicatorLines.push(debugLine);
 
         // uncomment 👇 this line to show helper lines for debugging
-        // debugLine.addTo(map);
+         //debugLine.addTo(map);
```

Comment:
Super small thing, but please revert this change ;-)

Reply:
done :)